src/containers/forgot-password/ForgotPasswordReset.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { useParams } from 'react-router-dom';
+import { Button, Form, Input, Typography } from 'antd';
+import authClient from '../../auth/authClient';
+
+const { Title } = Typography;
+
+const ForgotPasswordReset: React.FC = () => {
+  const { key } = useParams();
+
+  // gotta change this
+  const onFinish = (values: any) => {
+    authClient.forgotPasswordReset({
+      secretKey: key,
+      newPassword: values.password,
+    });
+  };
+  return (
+    <>
+      <Helmet>
+        <title>Forgot Password</title>
+        <meta name="description" content="Description goes here." />
+      </Helmet>
+      <div className="content-container">
+        <Title>Forgot Password</Title>
+        <Form name="basic" onFinish={onFinish}>
+          <Form.Item
+            label="Password"
+            name="password"
+            rules={[
+              { required: true, message: 'Please enter your new password!' },
+            ]}
+          >
+            <Input.Password />
+          </Form.Item>
+
+          <Form.Item
+            label="Confirm Password"
```

Comment:
Are we checking that both passwords entered are the same?

Reply:
Ah we should. I just added that in `Settings`

Also just noticed this, but please rename to `containers/forgotPassword/index.tsx`